com.unity.shadergraph/Documentation~/surface-options.md

```diff
@@ -49,7 +49,7 @@ This option's functionality corresponds to the command <a href="https://docs.uni
 </td></tr>
 
 
-
+<tr><td><b>Support VFX Graph</b></td><td colspan="2">This property is only available if the <a href="https://docs.unity3d.com/Packages/com.unity.visualeffectgraph@12.0/manual/">Visual Effect Graph package</a> is installed. Indicates whether this Shader Graph supports the Visual Effect Graph. If you enable this property, output contexts can use this Shader Graph to render particles. The internal setup that Shader Graph does to support visual effects happens when Unity imports the Shader Graph. This means that if you enable this property, but don't use the Shader Graph in a visual effect, there is no impact on performance. It only affects the Shader Graph import time.</td><td>Not applicable.</td></tr>
```

Comment:
Few questions: 

1. Are you saying that enabling this on a Shader Graph creates a 1:1 relationship with that Shader Graph and a VFX Graph, or just that whatever output contexts are the VFX Graph package can access the Shader Graph? 

Basically: specific VFX Graph file, or the package as a whole? 

2. This option only affects import times for importing a Shader Graph into a Visual Effect Graph?

Reply:
Hello,
This documentation has been copy/past from [HDRP](https://github.com/Unity-Technologies/Graphics/blob/5e335c0df9d04dabcbe29eb22352f7c5153496f0/com.unity.render-pipelines.high-definition/Documentation~/snippets/shader-properties/support-vfx-graph.md)

> Are you saying that enabling this on a Shader Graph creates a 1:1 relationship with that Shader Graph and a VFX Graph, or just that whatever output contexts are the VFX Graph package can access the Shader Graph?

The same shaderGraph can be used in several vfx and several output, so, if I'm undersand correctly the question, it's the second option.

> This option only affects import times for importing a Shader Graph into a Visual Effect Graph?

To be honest, I'm not sure we added this precision here, typically, it indicates that you can enable it without worry about the performance impact, it only adds some metadata while importing the shadergraph to allow the potential usage in VFX.
So, actually, it affects the import time of the ShaderGraph itself in the project even if it isn't used in any VFX.